README.md

```diff
@@ -17,11 +17,34 @@ npm install react-vapor
 > The project is at its early stages, some components can still have _lots_ of breaking changes between versions.
 
 ### Dependencies
-Those are required in order to use react-vapor in your own project:
-- [Tether](http://tether.io/)  (tested with 1.3.7)
-- [React](https://facebook.github.io/react/) (tested with 15.3.1)
-- [React DOM](https://facebook.github.io/react/) (tested with 15.3.1)
-- [Underscore](http://underscorejs.org/)  (tested with 1.8.3)
+React-Vapor is bundled with the popular bundler tool [Webpack](https://webpack.js.org/). However, we specify many dependencies as [externals](https://webpack.js.org/configuration/externals/#src/components/Sidebar/Sidebar.jsx) to maintain the library at a reasonable size, and to allow developers to cherry pick missing dependencies according to their own projects. The following packages are the external dependencies utilized by React-Vapor:      
```

Comment:
This is very picky of me and I acknowledge that this is really more of a personal preference (or a pet peeve in this case). So, I won't hold it against you if you do not change it, but I would change `utilized` for `used`. Thishttps://english.stackexchange.com/a/143945 explains my reasoning (especially the last box). But in the end we write code, not books, so as long as documentation is clear, it's fine by me.

Reply:
nice KT, will do

Reply:
👍